src/python/pants/backend/python/dependency_inference/module_mapper.py

```diff
@@ -317,12 +317,9 @@ async def map_module_to_address(
     request: PythonModuleOwnersRequest,
     first_party_mapping: FirstPartyPythonModuleMapping,
     third_party_mapping: ThirdPartyPythonModuleMapping,
-    python_setup: PythonSetup,
 ) -> PythonModuleOwners:
     providers = [
-        *third_party_mapping.providers_for_module(
-            request.module, resolve=request.resolve if python_setup.enable_resolves else None
-        ),
+        *third_party_mapping.providers_for_module(request.module, resolve=request.resolve),
```

Comment:
Note that `request.resolve` can still be `None` if callers set it to that. Currently only `python_distribution` dependency inference does that, which we believe is the correct behavior.